### ISSUE TITLE
Variables in Axiom queries do not respect panel repeat option

### ISSUE THREAD

**Issue by van-vliet** [axiomhq/axiom-grafana#69]
Hi

I have created a dashboard that pulls a list of environments from Axiom as a variable.
Then, for each environment it creates a panel that shows something for that environment.

My variable content looks like this:
<img width="284" alt="image" src="https://github.com/user-attachments/assets/dac4e861-d0cf-44d4-bc7c-0f306d1a6cd1">

My dashboard panel uses the repeat panel option over the `env` variable:
<img width="411" alt="image" src="https://github.com/user-attachments/assets/51c509da-f689-4896-8871-b091b13b0111">

When using the `${env}` variable in a repeated panel, the actual value of the variable becomes the panel specific value instead of the list of all environments. This is a default behaviour in most parts of Grafana. 
<img width="1791" alt="image" src="https://github.com/user-attachments/assets/7d92fa06-67db-4f8c-86d9-adbf7f766bb5">

Here is the Axiom query I use in the repeated panel:
```
['global']
| where ['resource.env'] == "${env}"
| where severity_number >= 17 // maps to error and fatal
| summarize count() by ['severity']
```

When I inspect the value of a repeat panel performing an Axiom query, it seems the entire list of environments is put into the query. I expected it to only insert the specific value of that panel:

<img width="1800" alt="image" src="https://github.com/user-attachments/assets/742d1142-fe86-4075-ab7d-c4b4e1facd54">

I compared this to the behaviour of another datasource - in this case Prometheus. I created a repeated panel and created a query using the same variable:
```
sum(kafka_topic_partitions{environment=~"${env}"})
```
I then checked the actual query send to the backend, and it only contains the specific panel value:
<img width="1800" alt="image" src="https://github.com/user-attachments/assets/0a03b353-60b8-4b06-b740-fb34a62ebeda">

Here is the dashboard I used to produce the error. It cannot completely reproduce the issue as it depends on both an Axiom and Prometheus datasource, but it will show you how my panels were configured.
[repeat-error-dashboard.json](https://github.com/user-attachments/files/16777137/repeat-error-dashboard.json)

Additional info:
The issue also happens when using the "table" panel type.

Versions used:
Grafana v11.1.4 (13173c9874)
Axiom plugin version: 0.4.0

**Comment by schehata** [comment]
hi @van-vliet, thanks for reporting this. I am looking into it at the moment, will get back to you.

**Comment by schehata** [comment]
@van-vliet I have released a new version (v0.5.1) with a fix for the mentioned issue, let me know if something is not right.

**Comment by van-vliet** [comment]
Hi @SollyzDev 
Thanks for the quick fix. Repeat panels now sent the correct query.
